src/main/java/net/dries007/tfc/util/LogFileWriter.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Work under Copyright. Licensed under the EUPL.
+ * See the project README.md and LICENSE.txt for more information.
+ */
+
+package net.dries007.tfc.util;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import net.dries007.tfc.TerraFirmaCraft;
+
+/**
+ * Handles writing text to a log file
+ * Makes code clean :D
```

Comment:
I disagree strongly - this seems way more code than is nessecary here. Writing a single bunch of stuff to a file, if buffered properly:
```java
List<String> lines;
try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
    for (String s : lines) {
        writer.write(s);
    }
} catch (IOException e) {
    // log or whatever
}
```